addons/hr_holidays/tests/test_company_leave.py

```diff
@@ -313,7 +313,7 @@ def test_leave_whole_company_07(self):
         })
         company_leave._compute_date_from_to()
 
-        with self.assertQueryCount(__system__=739, admin=865):
+        with self.assertQueryCount(__system__=842, admin=865):
```

Comment:
Well this is an issue. We cannot accept 110 additional queries in a side addon which has already been optimised.